perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -918,11 +919,11 @@ sub as_hashref {
     };
     $out->{is_fixed} = $self->fixed_states->{ $self->state } ? 1 : 0 if !$cols || $cols->{is_fixed};
     $out->{photos} = [ map { $_->{url} } @{$self->photos} ] if !$cols || $cols->{photos};
-    $out->{meta} = $self->confirmed ? $self->meta_line( $c->user ) : '' if !$cols || $cols->{meta};
-    $out->{created_pp} = $c->cobrand->prettify_dt( $self->created ) if !$cols || $cols->{created_pp};
+    $out->{meta} = $self->confirmed ? $self->meta_line : '' if !$cols || $cols->{meta};
```

Comment:
The removal of `$c->user` means the body contribution detail is omitted later on (I think?) - does that matter/is it intentional?

Reply:
Nothing uses the meta line, as far as I can tell. I thought it a bit odd that hashref output would include random extra user details here depending on who requested it, too, so I was happy for it to not do that.